lib/plug_body_digest.ex

```diff
@@ -100,20 +100,24 @@ defmodule PlugBodyDigest do
     algorithms = Keyword.get(opts, :algorithms, @algorithms)
 
     case verify(conn, algorithms) do
-      :ok ->
```

Comment:
This will result in `** (CaseClauseError) no case clause matching: :ok` in some cases when `verify/2` returns `:ok`